diff --git a/board/F+S/common/Kconfig b/board/F+S/common/Kconfig
index 5ae19fdaa..8b322fbcd 100644
--- a/board/F+S/common/Kconfig
+++ b/board/F+S/common/Kconfig
@@ -16,7 +16,7 @@ config FS_BOARD_COMMON
 
 config FS_BOARD_MODE_RO
 	bool "Load rootfs read-only (F&S)"
-	default y
+	default n
 	help
 	  F&S boards usually start the rootfs in read-only mode to
 	  avoid any data loss when the power fails or when the board is
