diff --git a/plat/imx/imx93/imx93_bl31_setup.c b/plat/imx/imx93/imx93_bl31_setup.c
index 7b7aa30f3..d48cd3d16 100644
--- a/plat/imx/imx93/imx93_bl31_setup.c
+++ b/plat/imx/imx93/imx93_bl31_setup.c
@@ -67,6 +67,7 @@ static uint32_t get_spsr_for_bl33_entry(void)
 void bl31_early_platform_setup2(u_register_t arg0, u_register_t arg1,
 		u_register_t arg2, u_register_t arg3)
 {
+#if DEBUG_CONSOLE
 	static console_t console;
 
 	console_lpuart_register(IMX_LPUART_BASE, IMX_BOOT_UART_CLK_IN_HZ,
@@ -74,6 +75,7 @@ void bl31_early_platform_setup2(u_register_t arg0, u_register_t arg1,
 
 	/* This console is only used for boot stage */
 	console_set_scope(&console, CONSOLE_FLAG_BOOT);
+#endif
 
 	/*
 	 * tell BL3-1 where the non-secure software image is located
diff --git a/plat/imx/imx93/include/platform_def.h b/plat/imx/imx93/include/platform_def.h
index 2d140400d..e36628850 100644
--- a/plat/imx/imx93/include/platform_def.h
+++ b/plat/imx/imx93/include/platform_def.h
@@ -53,7 +53,9 @@
 #define MAX_MMAP_REGIONS		16
 #endif
 
-#define IMX_LPUART_BASE			U(0x44380000)
+#define DEBUG_CONSOLE   0
+
+#define IMX_LPUART_BASE			U(0x44390000)
 #define IMX_BOOT_UART_CLK_IN_HZ		U(24000000) /* Select 24MHz oscillator */
 #define IMX_CONSOLE_BAUDRATE		115200
